Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+#!groovy
```

Comment:
Have you considered writing this with declarative pipelines? It does a good job at structuring what goes in a Jenkinsfile so people don't go wild adding tons of test logic here.